subxt/src/error/mod.rs

```diff
@@ -19,6 +19,7 @@ pub use dispatch_error::{
 
 // Re-expose the errors we use from other crates here:
 pub use crate::metadata::Metadata;
+pub use jsonrpsee_types::error::ErrorObject;
```

Comment:
you need to feature-gate this and I think this will work:

```suggestion
#[cfg(feature = "jsonrpsee")]
pub use jsonrpsee::types::error::ErrorObject;
```